tests/test_query.py

```diff
@@ -319,3 +273,141 @@ async def mock_get(
         "Subjects cannot both be healthy controls and have a diagnosis"
         in response.text
     )
+
+
+@pytest.mark.parametrize("valid_min_num_sessions", [1, 2, 4, 7])
+def test_get_valid_sensible_min_num_sessions(
```

Comment:
@rmanaem: can you rename this thing (and docstring) and remove "sensible" here as well, for consistency?